1.7/usage/managing-services/uninstall.md

```diff
@@ -32,6 +32,28 @@ Services can be uninstalled from either the web interface or the CLI. If the ser
 
 3.  Hover your cursor over the name of the package you wish to uninstall and you will see a red "Uninstall" link to the right. Click this link to uninstall the package.
 
+### Troubleshooting
+
+It's possible for an uninstall to fail with the following error message:
+
+```
+Incomplete uninstall of package [chronos] due to Mesos unavailability
+```
+
+In that case the service is inactive and will not appear in the UI, but can be found with the CLI:
+
+```bash
+$ dcos service --inactive
+NAME          HOST     ACTIVE  TASKS  CPU  MEM  DISK  ID
+chronos    10.0.6.138  False     0    0.0  0.0  0.0   7c0a7bd4-3649-4ec1-866c-5db8f2292bf2-0001
+```
+
+Perform a service shutdown with its ID to complete the uninstall operation (the framework cleaner script may still need to be run):
```

Comment:
`You can complete the uninstall by shutting down the service by using this CLI command with the service ID specified, and then run the [framework cleaner](#framework-cleaner):`